### ISSUE TITLE
ReferenceError: Card is not defined

### ISSUE THREAD

**Issue by budde377** [Dwlad90/stylex-swc-plugin#10]
Thank you for a great library.

I've encountered a bug when combining default and named exports of constants.

I have been able to reproduce it in your provided sandbox by modifying the `Card.tsx` file:

1. Add a named export (`export const styles =`).
2. Change the default exported component from a function to a constant.

See the code below.

```ts
import * as stylex from "@stylexjs/stylex";
import { globalTokens as $, spacing, text } from "./globalTokens.stylex";
import { colors } from "@stylexjs/open-props/lib/colors.stylex";
import { tokens } from "./CardTokens.stylex";

type Props = Readonly<{
  title: string;
  body: string;
  href: string;
}>;

type TMobile = "@media (max-width: 700px)";

const MOBILE = "@media (max-width: 700px)" satisfies TMobile;
const REDUCE_MOTION = "@media (prefers-reduced-motion: reduce)";

const cardBgTransparent = `rgba(${$.cardR}, ${$.cardG}, ${$.cardB}, 0)`;
const CardBgTranslucent = `rgba(${$.cardR}, ${$.cardG}, ${$.cardB}, 0.1)`;
const cardBorderTransparent = `rgba(${$.cardBorderR}, ${$.cardBorderG}, ${$.cardBorderB}, 0)`;
const cardBorderHover = `rgba(${$.cardBorderR}, ${$.cardBorderG}, ${$.cardBorderB}, 0.1)`;

export const styles = stylex.create({
  link: {
    display: {
      default: "flex",
      [MOBILE]: "block",
    },
    alignItems: "center",
    justifyContent: "flex-start",
    flexDirection: "column",
    borderRadius: spacing.xs,
    backgroundColor: {
      default: cardBgTransparent,
      ":hover": CardBgTranslucent,
    },
    borderWidth: 1,
    borderStyle: "solid",
    borderColor: {
      default: cardBorderTransparent,
      ":hover": cardBorderHover,
    },
    color: "inherit",
    fontFamily: $.fontSans,
    padding: spacing.sm,
    transitionProperty: "background-color, border-color",
    transitionDuration: "400ms",
    [tokens.arrowTransform]: {
      // eslint-disable-next-line @stylexjs/valid-styles
      default: "translateX(0)",
      // eslint-disable-next-line @stylexjs/valid-styles
      ":hover": "translateX(4px)",
    },
    textAlign: "center",
    textDecoration: "none",
  },
  h2: {
    color: colors.blue3,
    fontSize: text.h4,
    fontWeight: 600,
    marginBottom: {
      default: spacing.xs,
      [MOBILE]: spacing.xxs,
    },
  },
  span: {
    display: "inline-block",
    transitionProperty: "transform",
    transitionDuration: {
      default: "200ms",
      [REDUCE_MOTION]: "0s",
    },
    transform: tokens.arrowTransform,
  },
  p: {
    margin: 0,
    opacity: 0.6,
    fontSize: text.p,
    textWrap: "balance",
    lineHeight: 1.5,
    maxWidth: "30ch",
  },
  dynamic: (color: string) => ({
    color: color,
  }),
});

const Card = ({ title, body, href }: Props) => {
  return (
    <a
      href={href}
      {...stylex.props(styles.link)}
      target="_blank"
      rel="noopener noreferrer"
    >
      <h2 {...stylex.props(styles.h2)}>
        {title} <span {...stylex.props(styles.span)}>→</span>
      </h2>
      <p {...stylex.props(styles.p)}>{body}</p>
    </a>
  );
};

export default Card;
```

When inspecting the generated output it would seem that the `const Card` assignment is stripped from the output code.

([sandbox](https://codesandbox.io/p/github/Dwlad90/nextjs-app-dir-stylex/csb-vp7scd/draft/zealous-elion?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522cm0l2r95400063b6jzr0m2xki%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522cm0l2r95400023b6jgjqilr8p%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522cm0l2r95400043b6jl0hxp6w3%2522%257D%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522cm0l2r95400053b6j4soh24kf%2522%257D%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522cm0l2r95400023b6jgjqilr8p%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm0l2r95400013b6jbmis2wve%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252FREADME.md%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%252C%257B%2522id%2522%253A%2522cm0l7kz9o001h3b6js27ku16z%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A1%252C%2522startColumn%2522%253A1%252C%2522endLineNumber%2522%253A1%252C%2522endColumn%2522%253A1%257D%255D%252C%2522filepath%2522%253A%2522%252Fapp%252Flayout.tsx%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%252C%257B%2522id%2522%253A%2522cm0m6n03b00023b6jt28n6aaq%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A42%252C%2522startColumn%2522%253A25%252C%2522endLineNumber%2522%253A42%252C%2522endColumn%2522%253A25%257D%255D%252C%2522filepath%2522%253A%2522%252Fapp%252FCard.tsx%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522id%2522%253A%2522cm0l2r95400023b6jgjqilr8p%2522%252C%2522activeTabId%2522%253A%2522cm0m6n03b00023b6jt28n6aaq%2522%257D%252C%2522cm0l2r95400053b6j4soh24kf%2522%253A%257B%2522id%2522%253A%2522cm0l2r95400053b6j4soh24kf%2522%252C%2522activeTabId%2522%253A%2522cm0m6nwah00103b6jhwjl591l%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A2222%252C%2522id%2522%253A%2522cm0l2rsep00103b6j42dup5ne%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%252C%257B%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A3000%252C%2522id%2522%253A%2522cm0l7k22c000d3b6j9h9tc9gr%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%252C%257B%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A3000%252C%2522id%2522%253A%2522cm0m6nwah00103b6jhwjl591l%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%255D%257D%252C%2522cm0l2r95400043b6jl0hxp6w3%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm0l2r95400033b6jlxy80qz7%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522dev%2522%257D%255D%252C%2522id%2522%253A%2522cm0l2r95400043b6jl0hxp6w3%2522%252C%2522activeTabId%2522%253A%2522cm0l2r95400033b6jlxy80qz7%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D))

**Comment by durkhallow** [comment]
I am also seeing this on [another sandbox example](https://codesandbox.io/p/devbox/nff776?file=%2F%40apps%2Fweb%2Fsrc%2Fapp%2Fpage.tsx%3A18%2C1)

**Comment by Dwlad90** [comment]
A new release of the plugin has been uploaded to NPM.

The fix has been merged into PR #14

**Comment by Dwlad90** [comment]
Fixed #14

**Comment by Dwlad90** [comment]
Fixed #14
